src/tests/unittests/configuration_helper/test_instrument_adapter_factory.py

```diff
@@ -36,33 +40,11 @@ def test_raise_value_error(self):
         with self.assertRaises(ValueError):
             InstrumentAdapterFactory.get_instrument_adapter('SomeAdapter', 'dev42')
 
-    def test_import_error_as_value_error(self):
-        import builtins
-        import sys
-        import importlib
-        importlib.invalidate_caches()
+    def test_external_adapters(self):
+        self.assertRaises(ValueError, InstrumentAdapterFactory.get_instrument_adapter, 'DummyInstrumentAdapter', '')
```

Comment:
maybe make a separate test case for this assertion

Reply:
Why? It's part of `InstrumentAdapterFactory`. And if still, what would be your suggestion on how to implement this? Within `test_instrument_adapter_factory` as a separate test case or within another file?